pmm-app/src/pmm-settings/components/PlatformLogin/PlatformLogin.styles.ts

```diff
@@ -44,6 +44,7 @@ export const getStyles = stylesFactory((theme: GrafanaTheme) => ({
   `,
   form: css`
     max-width: 300px;
+    min-width: 300px;
```

Comment:
I would do something like `min-width: 150px` instead here and use `flex: 1` to `tabContentWrapper` in `Settings.styles.ts`, in order make this shrink a bit on smaller screens